engine/Default/forces_drop_processing.php

```diff
@@ -132,7 +132,11 @@
 }
 
 // message to send out
-if ($forces->getOwnerID() != $player->getAccountID() && $forces->getOwner()->isForceDropMessages()) {
+// only send the message if the stack has (or had) scout drones
+if ($forces->getOwnerID() != $player->getAccountID() &&
+    $forces->getOwner()->isForceDropMessages() &&
+    ($forces->hasSDs() || $change_scout_drones != 0))
```

Comment:
Sounds good to me. I think what I'll do here is change the message back to being sent by the player (rather than "Scout Drone"), so that it makes more sense to send a message regardless of whether scout drones are present.